app/migrations/versions/961ab9adc300_.py

```diff
@@ -26,9 +25,15 @@ def upgrade():
     op.add_column('organisation', sa.Column('external_auth_username', sa.String(), nullable=True))
     op.add_column('organisation', sa.Column('_external_auth_password', sa.String(), nullable=True))
 
-    for org in session.query(Organisation).execution_options(show_all=True).all():
+    tcr = sa.sql.table('organisation',
+                       sa.Column('id', sa.Integer, primary_key=True),
+                       sa.Column('external_auth_username', sa.String(), nullable=True),
+                       sa.Column('_external_auth_password', sa.String(), nullable=True),
+                       sa.Column('name', sa.String(), nullable=True))
```

Comment:
I wasn't able to update the organisation model as the migration from a clean database was failing here, which makes sense in hindsight.

> The migration defines separate models because the models in your code represent the current state of the database, while the migrations represent steps along the way. Your database might be in any state along that path, so the models might not sync up with the database yet. Unless you're very careful, using the real models directly will cause problems with missing columns, invalid data, etc. It's clearer to explicitly state exactly what columns and models you will use in the migration.
https://stackoverflow.com/questions/24612395/how-do-i-execute-inserts-and-updates-in-an-alembic-upgrade-script/24623979

@michielderoos can you please triple check this works! 🙏 📿

Reply:
Also for the user migration https://github.com/teamsempo/SempoBlockchain/blob/50b6113f2ecd01749eb5ffc849299a2cc37f5771/app/migrations/versions/87b723e167d3_.py#L25-L34

Reply:
@tristanhcole Seems so obvious in hindsight!

Reply:
Will dig in and QA all the permutations here (Fresh DB/ unmigrated DB etc...)